inc/ThirdParty/Themes/Divi.php

```diff
@@ -108,6 +113,29 @@ public function add_divi_to_description( $disable_youtube_lazyload ) {
 		return $disable_youtube_lazyload;
 	}
 
+	/**
+	 * Disables setting explicit dimensions on images where Divi calculates height as percentage.
+	 *
+	 * @since 3.8.2
+	 *
+	 * @param array $images The array of images selected for adding image dimensions.
+	 *
+	 * @return array The array without images using data-height-percentage.
+	 */
+	public function disable_image_dimensions_height_percentage( array $images ) {
+		if ( ! self::is_divi() ) {
```

Comment:
why this checking here?
I can see that this callback won't be called from `get_subscribed_events` method if it's not Divi, I think no need to use it here.

Reply:
This is called from `get_subscribed_events()` on L:51, above.
If not guarded here, the testcase for leaving things alone for other themes will fail -- see this testcase:
https://github.com/wp-media/wp-rocket/blob/76838b53857ced21499de0869953b6f459e70aa0/tests/Fixtures/inc/ThirdParty/Themes/Divi/disableImageDimensionsHeightPercentage.php#L32